temporal-sdk/src/main/java/io/temporal/client/WorkflowInvocationHandler.java

```diff
@@ -490,7 +490,7 @@ public void invoke(
           throw new IllegalArgumentException(
               "Method '" + method.getName() + "' is not a WorkflowMethod");
         }
-        this.operation.prepareStart(untyped);
+        this.operation.prepareStart(untyped, args);
         state = State.UPDATE_RECEIVED;
```

Comment:
Do these enum names make sense? isn't the state now `START_RECEIVED`?

Reply:
Yup, I noticed this was wrong, too. It's fixed in https://github.com/temporalio/sdk-java/pull/2288/files#diff-62667c49d1f873a13d0894606bc2163f4db202d4f351c83bc1cf8c141e9342a8R481